src/main/java/com/spotify/docker/client/messages/Event.java

```diff
@@ -91,7 +143,50 @@ public String toString() {
         .add("status", status)
         .add("id", id)
         .add("from", from)
+        .add("type", type)
+        .add("action", action)
+        .add("actor", actor)
         .add("time", time)
+        .add("timeNano", timeNano)
         .toString();
   }
+
+  public static class Actor {
+    @JsonProperty("ID") private String id;
+    @JsonProperty("Attributes") private ImmutableMap<String, Object> attributes;
```

Comment:
It looks like it can be `<String, String>`. I'm not sure why I thought it was possible to get an `Object` there, but their [swagger.yaml](https://github.com/docker/docker/blob/master/api/swagger.yaml#L5336) says `Attributes` will always have `String` values.

I'll fix that.